Eatery/Views/Eateries/MealItemTableViewCell.swift

```diff
@@ -21,12 +29,59 @@ class MealItemTableViewCell: UITableViewCell {
         addSubview(nameLabel)
         nameLabel.translatesAutoresizingMaskIntoConstraints = false
         nameLabel.snp.makeConstraints { make in
-            make.edges.equalTo(self).inset(UIEdgeInsets(top: 8, left: 16, bottom: 8, right: 16))
+            make.edges.equalTo(self).inset(UIEdgeInsets(top: 4, left: 16, bottom: 4, right: 16))
         }
     }
 
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    /// Configures the cell's nameLabel text to match the menu item text (adjusting for whether or not the item is healthy) and adds a favorite button to the cell corresponding to whether the user has favorited the menu item
+    func configure(for menuItem: Menu.Item) {
+        self.menuItem = menuItem
+        
+        let itemText = (menuItem.healthy) ? NSMutableAttributedString(string: "\(menuItem.name.trim()) ")
+        .appendImage(UIImage(named: "appleIcon")!, yOffset: -1.5) : NSMutableAttributedString(string: menuItem.name)
```

Comment:
did you mean to do `menuItem.name.trim()` for both cases? also this is kind of hard to read, lets do:
```
let itemText = menuItem.healthy 
  ? NSMutableAttributedString(string: "\(menuItem.name.trim()) ")
       .appendImage(UIImage(named: "appleIcon"), yOffset: -1.5) 
  : NSMutableAttributedString(string: menuItem.name)
```